invenio_opendata/base/templates/about_cms.html

```diff
@@ -24,7 +24,15 @@ <h2 id="tools">Data and analysis tools</h2>
               <ul class="col-md-12">
                 <li>Downloadable datasets
                   <ul class="col-md-12">
-                    <li><a href="{{ url_for('.collection', name = 'CMS-Primary-Datasets') }}">Primary datasets</a>: full reconstructed data with no other selections. The data here are referred to as "reconstructed data": fragmented data from various sub-detectors are processed or "reconstructed" to provide coherent information about individual <a href="{{ url_for('.about_physics') }}">physics objects</a> such as electrons or particle jets</li>
+                    <li><a href="{{ url_for('.collection', name = 'CMS-Primary-Datasets') }}">Primary datasets</a>:
+                        <ol>
+                            <li>
+                                full reconstructed collision data with no other selections. The data here are referred to as "reconstructed data": fragmented data from various sub-detectors are processed or "reconstructed" to provide coherent information about individual <a href="{{ url_for('.about_physics') }}">physics objects</a> such as electrons or particle jets
+                            </li>
+                            <li>
+                                simulation data (for data from 2011)
```

Comment:
@RaoOfPhysics @katilp FWIW I think it might be confusing that here, in the help file, we are using the notion of "primary datasets" as being composed of "collision data" and "simulation data", while elsewhere, in the collection tree, we have defined two separate collections "CMS Primary Datasets" and "CMS Simulated Datasets".  It would be good to synchronise the notions everywhere...